packages/chain-agnostic-permission/README.md

```diff
@@ -0,0 +1,15 @@
+# `@metamask/chain-agnostic-permission`
+
+Defines a permission caveat and endowment designed to match [CAIP-25](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-25.md) and related specifications.
```

Comment:
Does this sound right?

Reply:
not sure if `match` is necessarily the right word. Perhaps `similar in structure to ...`?

Reply:
Maybe "mimic" instead of "match"?

Reply:
Updated here: [dc3db6f](https://github.com/MetaMask/core/pull/5476/commits/dc3db6f5c691fa7099d678e440fac75ab8c32a85)